ml-agents/mlagents/trainers/settings.py

```diff
@@ -183,6 +183,7 @@ def to_settings(self) -> type:
 class RewardSignalSettings:
     gamma: float = 0.99
     strength: float = 1.0
+    normalize: bool = False
```

Comment:
We should put this in GAILRewardSignalSettings for now, until we implement it in Curiosity as well.

Reply:
resolved in https://github.com/Unity-Technologies/ml-agents/pull/4982